lib/config/meta.go

```diff
@@ -74,7 +74,7 @@ type TSMeta struct {
 	Spdy    Spdy    `toml:"spdy"`
 
 	// TLS provides configuration options for all https endpoints.
-	TLS *tls.Config `toml:"-"`
+	TLS tlsconfig.Config `toml:"tls"`
```

Comment:
can we still use standard library tlsconfig?

Reply:
`tlsconfig` is a wrapper of the standard library `tls.Config`, which is used to mapping the configuration file. After reading the configuration file, use `Parse` to get `tls.Config`.